api/models/topic.models.js

```diff
@@ -0,0 +1,9 @@
+const connection = require("../../db/connection");
+
+exports.fetchAllTopics = () => {
+  let queryString = "SELECT * from topics";
```

Comment:
currently (and as far as you know) this doesn't change so we should really use a const.